src/main/java/it/gov/pagopa/payhub/activities/activity/utility/SendEmailIngestionFlowActivityImpl.java

```diff
@@ -0,0 +1,146 @@
+     */
+    @Override
+    public boolean sendEmail(Long ingestionFlowFileId, boolean success) {
+
+        try {
+            IngestionFlowFileDTO ingestionFlowFileDTO = ingestionFlowFileDao.findById(ingestionFlowFileId)
+                    .orElseThrow(() -> new IngestionFlowNotFoundException("Cannot found ingestionFlow having id: "+ ingestionFlowFileId));
+            String ipaCode = ingestionFlowFileDTO.getOrg().getIpaCode();
+            UserInfoDTO userInfoDTO = userAuthorizationService.getUserInfo(ipaCode, ingestionFlowFileDTO.getOperatorName());
+            OrganizationDTO organizationDTO = organizationAuthorizationService.getOrganizationInfo(ipaCode);
+            MailTo mailTo = getMailFromIngestionFlow(ingestionFlowFileDTO, success);
+            mailTo.setTo(new String[]{userInfoDTO.getEmail()});
+            if (organizationDTO!= null && StringUtils.isNotBlank(organizationDTO.getAdminEmail()) &&
+                ! organizationDTO.getAdminEmail().equalsIgnoreCase(userInfoDTO.getEmail())) {
+                    mailTo.setCc(new String[]{organizationDTO.getAdminEmail()});
+            }
+            sendMailService.sendMail(mailTo);
+        }
+        catch (Exception e){
+            log.error("Sending mail failed", e);
+            return false;
+        }
+        return true;
+    }
+
+    private MailTo getMailFromIngestionFlow(IngestionFlowFileDTO ingestionFlowFileDTO, boolean success) throws Exception {
+        String flowType = ingestionFlowFileDTO.getFlowFileType();
+        if (! flowType.equalsIgnoreCase("R")) {
+            log.error("Sending e-mail not supported for flow type: {}", flowType);
+            throw new IngestionFlowTypeNotSupportedException("Sending e-mail not supported for flow type: "+flowType);
+        }
+
+        String subject = (success
+                ? emailTemplatesConfiguration.getPaymentsReportingFlowOk()
+                : emailTemplatesConfiguration.getPaymentsReportingFlowKo()
+                ).getSubject();
+        String body = (success
+                ? emailTemplatesConfiguration.getPaymentsReportingFlowOk()
+                : emailTemplatesConfiguration.getPaymentsReportingFlowKo()
+                ).getBody() ;
+
+
+        MailTo mailTo = new MailTo();
+        mailTo.setMailSubject(StringSubstitutor.replace(subject, mailTo.getParams(), "{", "}"));
+        String htmlText = StringSubstitutor.replace(body, mailTo.getParams(), "{", "}");
+        String plainText = Jsoup.clean(htmlText, "", Safelist.none(), new Document.OutputSettings().prettyPrint(false));
+        mailTo.setHtmlText(plainText);
+        mailTo.setParams(getMailParameters(ingestionFlowFileDTO, body));
+        mailTo.setMailSubject(subject);
+
+        if (StringUtils.isNotBlank(ingestionFlowFileDTO.getDiscardedFileName()) &&
+                StringUtils.isNotBlank(ingestionFlowFileDTO.getFilePathName()))  {
+            mailTo.setAttachmentPath(
```

Comment:
I think it's necessary to trace PU has sent an email with personal data. Later we could think to choose another approach to manage it.

Reply:
This file is supposed to be ciphered... Thus we cannot simply attach it, it would be not readable...

MyPay actually is not sending any data, it's providing an URL to download the file instead (the download service will decipher it).
The fastest way I see is actually to no provided any file: the task on JIRA would the reminder on handling it